schema/workflow.yaml

```diff
@@ -1188,13 +1188,21 @@ $defs:
             title: ExpressionErrorInstance
             description: An expression based error instance.
       title:
-        type: string
-        title: ErrorTitle
         description: A short, human-readable summary of the error.
+        title: ErrorTitle
+        anyOf:
```

Comment:
Should be `oneOf` AFAIK ;)

Reply:
@cdavernas that was my first try, see the first commit. But the validation was failing. And once I get there: https://stackoverflow.com/a/34038277/1354502. I figured `anyOf` would be the right approach. 

Also, we have this in the schema too: https://github.com/serverlessworkflow/specification/blob/main/schema/workflow.yaml#L1220-L1230

Which is a similar context.

If you have any ideas why, we can change the type no problem.

Reply:
@ricardozanini I guess that's because of a multi match. Second alternative also matches the first. My bad!